src/main/java/com/example/enterprise/dao/ExpenseDAO.java

```diff
@@ -11,11 +11,20 @@
 
     List<Expense> allExpenses = new ArrayList<>();
 
+    /**
+     * Save an expense to the allExpenses arraylist
+     * @param expense an expense given by user
+     * @throws Exception
+     */
     @Override
     public void save(Expense expense) throws Exception {
         allExpenses.add(expense);
     }
 
+    /**
+     * List all entered expenses in allExpenses arraylist
+     * @return allExpense arraylist
+     */
```

Comment:
Added some java doc for the methods with meaningful comments to explain the logic of the method. Java doc is necessary and a good coding convention to keep your code clean. 
To learn more about JavaDoc here is the link: https://www.tutorialspoint.com/java/java_documentation.htm